dapr/proto/runtime/v1/dapr.proto

```diff
@@ -680,13 +678,13 @@ message UnlockResponse {
 }
 
 message WorkflowReference {
-  string instance_id = 1;
+  string instance_id = 1 [json_name = "instanceId"];
 }
 
 message GetWorkflowRequest {
-  string instance_id  = 1;
-  string workflow_name = 2;
-  string workflow_component = 3;
+  string instance_id  = 1 [json_name = "instanceId"];
+  string workflow_component = 2 [json_name = "workflowComponent"];
```

Comment:
Reviewers: I swapped workflow_component and workflow_name so in all messages, component is 2 and name is 3. This is technically a breaking change, but the entire API underwent significant breaking changes already.

Reply:
is there a reason for this switch?

Reply:
Consistency because everywhere else name was 3 and component was 2. And this is a good time to add a breaking change since there's a bunch already.